text_handler.py

```diff
@@ -30,10 +47,13 @@ async def __handleYoutubeDownload(self, message: types.Message):
             filesize = stream.filesize
             telegram_audio_limit = 52428800
             if (filesize < telegram_audio_limit and length < 600):
-                output_filename = stream.download("downloads")
-                with open(output_filename, "rb") as f:
-                    await message.reply_audio(f, duration=length, performer=author, title=title)
-                os.remove(output_filename)
+                output_audiofile = stream.download("downloads")
+                output_thumbnail = "downloads/thumbnail.jpg"
+                mkthumb(yt.thumbnail_url, output_thumbnail)
+                with open(output_audiofile, "rb") as a, open(output_thumbnail, "rb") as t:
```

Comment:
can you change ```a``` and ```t``` to ```audio``` and ```thumb```? the code will be a little more readable this way. I think it's fine if you format message.reply_audio like this:
```
                    await message.reply_audio(audio, 
                                              duration=length, 
                                              performer=author, 
                                              title=title, 
                                              thumb=thumb)
```